README.md

```
@@ -1 +1,241 @@
-# 27th-App-Team-1-iOS
+# NDGL-iOS
+
+<p align="center">
+  <img src="https://img.shields.io/badge/Swift-5.0-orange.svg" />
```

markdownlint-cli2 (0.18.1)

Images should have alternate text (alt text)

(MD045, no-alt-text)